functions.md

```diff
@@ -526,6 +526,25 @@ trim(false)        => "false"
 trim(null)         => null
 ```
 
+### _uuid(long, long) -> string_
+
+Generates a with dashes formatted UUID string. 
+If no parameter are passed, a random UUID version 4 will be generated.
+If one parameter is passed, a UUID version 1 is generated with the MSB part based on that value and the LSB is randomly generated.
```

Comment:
I think it's a good idea to spell out LSB and MSB here for people who don't know what you mean.